engine/include/vull/test/matchers.hh

```diff
@@ -59,7 +59,10 @@ class EqualTo {
 public:
     explicit EqualTo(const T &expected) : m_expected(expected) {}
 
-    void describe(Message &message) const { message.append_value(m_expected); }
+    void describe(Message &message) const {
+        message.append_text("is equal to ");
```

Comment:
```suggestion
        message.append_text("equal to ");
```
I think we want to drop the `is` here, otherwise `is(equal_to(5))` would print as `is is equal to 5`.